### PR TITLE
feat: ask for model in init workflow

### PR DESCRIPTION
**Pull request by ankitjena**
## Major
During `graphback init` ask if user want's to include model or not. If yes, include one of the example models, if not create a file with commented out text

### Minor
Fix indentation in template literals(new line strings)

### CODE REVIEW

File: packages/graphback-cli/src/templates/modelTemplates.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import { writeFileSync } from 'fs';
+import { prompt as ask } from 'inquirer'
+import ora from 'ora'
+import { Model } from './templateMetadata'
+
+export const defaultModel: string = `# type Note {
```
Comment (wtrocki) [inline packages/graphback-cli/src/templates/modelTemplates.ts]: Would it be possible to have them as separate files. each as separate graphql file. We can read them directly using fs

File: packages/graphback-cli/src/templates/modelTemplates.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    content: `type Note {
+  id: ID!
+  title: String!
+  description: String!
+  ## Relationship
+  comment: [Comment]!
+
+}
+
+type Comment {
```
Comment (wtrocki) [inline packages/graphback-cli/src/templates/modelTemplates.ts]: love templates great ideas

File: packages/graphback-cli/src/templates/modelTemplates.ts [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        type: 'list',
+        name: 'modelName',
+        message: 'Choose one of the example models',
+        choices: allModels.map((m: Model) => m.name)
+      }
+    ])
+
+    const content = allModels.find((m: Model) => m.name===modelName).content
+    
+    return [modelName, content]
```
Comment (wtrocki) [inline packages/graphback-cli/src/templates/modelTemplates.ts]: How about default

File: packages/graphback-cli/src/templates/templateMetadata.ts [hunk 4]
```diff
@@ -8,4 +8,9 @@ export interface TemplateRepository {
   uri: string
   branch: string
   path: string
+}
+
+export interface Model {
```
Comment (wtrocki) [inline packages/graphback-cli/src/templates/templateMetadata.ts]: lack of comments. Rename to make it more clear what this is. ExampleTypes ?

File: packages/graphback-cli/src/templates/modelTemplates.ts (hunk 1 above)
Comment (ankitjena) [reply packages/graphback-cli/src/templates/modelTemplates.ts]: We would have to copy them in build then, Examples aren't that large anyways

**Pull request merged**